openfisca_aotearoa/situation_examples/single.json

```diff
@@ -15,3 +15,4 @@
         }
     }
 }
+
```

Comment:
It would probably be useful to either update of delete that file, as it is currently misleading.